src/main/java/de/orchound/gameoflife/painting/FillPainter.java

```diff
@@ -0,0 +1,41 @@
+package de.orchound.gameoflife.painting;
+
+import de.orchound.gameoflife.model.Game;
+import org.joml.Vector2i;
+import org.joml.Vector2ic;
+
+public class FillPainter implements Painter {
+
+	private final Game game;
+	private boolean currentlyPainting = false;
+	private boolean paintMode = false;
+
+	public FillPainter(Game game) {
+		this.game = game;
+	}
+
+	private void fillCells(Vector2ic cell) {
+		if (game.getCellStatus(cell) != paintMode && game.cellInBoardRange(cell)) {
+			game.setCellSilently(cell, paintMode);
+			fillCells(new Vector2i(cell.x(), cell.y()+1));
+			fillCells(new Vector2i(cell.x(), cell.y()-1));
```

Comment:
Fix operator spaces.
```suggestion
			fillCells(new Vector2i(cell.x(), cell.y() + 1));
			fillCells(new Vector2i(cell.x(), cell.y() - 1));
```